lib/ansible/modules/source_control/github_organization.py

```
@@ -0,0 +1,308 @@
+        'private_repo_count': org['total_private_repos'],
+        'default_repo_permission': org['default_repository_permission']
+    })
+    return res
+
+
+def create_organization(module, auth_header):
+    result = {'changed': False}
+
+    organization_name = module.params['name']
+    base_url = _get_base_url(module)
+
+    existing_org = _get_organization(module, base_url, auth_header, organization_name)
+
+    if existing_org is None:
+        url = _build_api_url(base_url, 'admin/organizations')
+        headers = {'Authorization': auth_header}
+        data = {
+            "login": organization_name,
+            # "profile_name": organization_name,
+            "admin": module.params['owner']
+        }
+
+        r, info = fetch_url(module,
+                            url,
+                            data=module.jsonify(data),
+                            method="POST",
+                            headers=headers)
+        if info['status'] == 201:
+            org = _get_organization(module, base_url, auth_header, organization_name)
+            result.update({'changed': True})
+            _merge_organization_result(result, org)
+        else:
+            failure = module.from_json(info['body'])
+            module.fail_json(msg='Failed to create organization: {0}'.format(failure['message']))
+    else:
+        _merge_organization_result(result, existing_org)
+
+    return result
+
+
+def rename_organization(module, auth_header):
+    result = {'changed': False}
+
+    organization_name = module.params['name']
+    new_name = module.params['rename_to']
+
+    if new_name is None:
+        module.fail_json(msg="A name to rename the organization to must be provided")
+
+    base_url = _get_base_url(module)
+    data = {'login': new_name}
+
+    existing_org = _get_organization(module, base_url, auth_header, organization_name)
+    target_org = _get_organization(module, base_url, auth_header, new_name)
+
+    if existing_org is not None:
```

If the organization being renamed doesn't exist yet, that should probably throw an error or a warning or something.